REUSE.toml

```
@@ -13,3 +13,8 @@ SPDX-License-Identifier = "MIT"
 path = ["test/test_data/**",]
 SPDX-FileCopyrightText = "The technology-data authors"
 SPDX-License-Identifier = "CC-BY-4.0"
+
+[[annotations]]
+path = ["src/technologydata/package_data/raw/*","src/technologydata/package_data/dea_energy_storage/*.json"]
+SPDX-FileCopyrightText = "The technology-data authors"
+SPDX-License-Identifier = "CC-BY-4.0"
```

```suggestion
[[annotations]]
path = [
    "src/technologydata/package_data/dea_energy_storage/*.json"
]
SPDX-FileCopyrightText = "The technology-data authors"
SPDX-License-Identifier = "CC-BY-4.0"
```

The file in `src/technologydata/package_data/raw/` is not copyright by us and requires a dedicated entry